src/ZuriAC.php

```diff
@@ -123,8 +123,13 @@ public function onLoad() : void {
 		self::$instance = $this;
 		ConfigManager::checkConfig();
 
+                if (!version_compare(PHP_VERSION, '8.3.0', '>=')) {
+                    $this->getLogger()->error("⚠️ You're running PHP " . PHP_VERSION . ", which is older than 8.3.0. Please upgrade your PHP Installion to 8.3.0 or later!");
```

Comment:
I think it would be better if we attach the download link for the appropriate PHP version.